### PR TITLE
Add Recaptcha to Feedback Form

### PR DESCRIPTION
**Pull request by JPrevost**
What
- adds recaptcha to feedback form
- uses test keys in pr build

Why
- to prevent spam

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -56,6 +56,8 @@
     "RAILS_SERVE_STATIC_FILES": {
       "required": true
     },
+    "RECAPTCHA_SECRET_KEY": "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe",
```
Comment (frrrances) [inline app.json]: is this ok to have in a public repo?

Comment (JPrevost) [inline app.json]: Thanks for checking! But yes, those are special "test keys" google provides for this purpose.

Comment (frrrances) [inline app.json]: ah that explains why it let me off so easy. clearly i am a robot.

Comment (JPrevost) [inline app.json]: :)

More info if you care:
https://developers.google.com/recaptcha/docs/faq

**Pull request merged**